Remove the VERSION file to fix Windows/MacOS builds.
On case-insensitive filesystems, MSVC may resolve #include <version>
to this data file instead of the C++ standard library header.
The original problem is in `rules_cc`. This workaround is only needed for
older version of `rules_cc` that don't include the fix in this PR: https://github.com/bazelbuild/rules_cc/pull/802

--- VERSION
+++ /dev/null
@@ -1 +0,0 @@
-4.7.2
